sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/SamplingResult.java

```diff
@@ -60,13 +61,50 @@ static SamplingResult create(SamplingDecision decision) {
    * @return A {@link SamplingResult} with the attributes equivalent to {@code attributes} and the
    *     provided {@code decision}.
    */
+  @SuppressWarnings("UngroupedOverloads")
```

Comment:
It would be better if we can make sure we group the overloads so we don't need this warning.

Reply:
I have removed them, could you please trigger the check for me again. Thanks~